src/main/java/com/redhat/rcm/offliner/Options.java

```diff
@@ -27,6 +27,8 @@
 public class Options
 {
 
+    public static final String DEFAULT_TYPE_MAPPING = "test-jar=jar:tests;ejb=jar;ejb-client=jar:client;maven-plugin=jar;java-source=jar:sources;javadoc=jar:javadoc";
+
```

Comment:
True, a properties file would be probably better, because even the default value is too long.
Not sure about the default value printing though. I'm afraid that the number of command-line parameters would be too high and since we move to properties file it would be pretty easy to extract the default props.